x-pack/platform/plugins/private/translations/translations/fr-FR.json

```diff
@@ -2657,7 +2657,6 @@
     "discover.badge.readOnly.text": "Lecture seule",
     "discover.badge.readOnly.tooltip": "Impossible d'enregistrer les sessions Discover",
     "discover.context.breadcrumb": "Documents relatifs",
-    "discover.context.contextOfTitle": "Les documents relatifs à #{anchorId}",
```

Comment:
It was deleted by `node ./scripts/i18n_check.js --fix` while the composition of the message was a bit different (`FormattedMessage` and `span` with anchor where siblings). I have a hunch that these should be reverted, because the copy remains the same as previously, but I'd appreciate @jughosta or @davismcphee to confirm 🙏

Reply:
Thanks @jughosta!